Themes/default/Agreement.template.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Simple Machines Forum (SMF)
+ *
+ * @package SMF
+ * @author Simple Machines
+ * @copyright 2018 Simple Machines
+ * @license http://www.simplemachines.org/about/smf/license.php BSD
+ *
+ * @version SMF 2.1 RC2
+ */
+
+// The main sub template - show the agreement and/or privacy policy
+function template_main()
+{
+	global $context, $scripturl, $txt;
+
+	if (!empty($context['accept_doc']))
+		echo '
+	<form action="', $scripturl, '?action=acceptagreement;doc=', $context['accept_doc'], '" method="post">';
+
+	if (!empty($context['agreement']))
+	{
+		echo '
+		<div class="cat_bar">
+			<h3 class="catbg">', $txt['agreement' . ($context['can_accept_agreement'] ? '_updated' : '')], '</h3>
+		</div>';
+
+		if ($context['can_accept_agreement'])
+		{
+			echo '
+		<p class="description">
+			', $txt['agreement_updated_desc'], '
+		</p>';
+		}
+		elseif (!empty($context['agreement_accepted_date']))
+		{
+			echo '
+		<p class="description">
+			', sprintf($txt['agreement_accepted'], timeformat($context['agreement_accepted_date'], false)), '
+		</p>';
+		}
+
+		echo '
+		<div class="roundframe">
+			<div>', $context['agreement'], '</div>
+		</div>';
+	}
+
+	if (!empty($context['policy']))
+	{
+		if (!empty($context['agreement']))
+			echo '<br />';
```

Comment:
Invalid HTML5 :) you can leave this br tag out completely because block elements and proper padding/margin should be enough to create space.